### ISSUE TITLE
When is v3.1 available from the logstash-plugin update utility?

### ISSUE THREAD

**Issue by tpnoonan** [logstash-plugins/logstash-codec-netflow#38]
Sorry all, I'm new to logstash and especially this netflow code.  I see that Jorrit release v3.1 

https://github.com/logstash-plugins/logstash-codec-netflow/releases/tag/v3.1.0

however, i'm having a hell of a time getting a centos os to build the source package.  What my question is however is about process.  If Jorrit releases a new version, when or how long does it take to be included "where ever" the process "bin/logstash-plugin update logstash-codec-netflow".  When i run that command now, i get that it's going to try and update the codec to v3.0 instead.

Thanks for any guidance.

**Comment by tpnoonan** [comment]
ok, posted a similar message on the log stash community and i see that the code needs to be pushed to rubygems.org

Jorrit, can you push?

**Comment by suyograo** [comment]
@tpnoonan 3.1.0 is only supported for LS 5.x release. You can install this using `bin/logstash-plugin install --version 3.1.0 logstash-codec-netflow`

@jorritfolmer Can you backport the IPFIX support to this branch `plugins-api-v1`? https://github.com/logstash-plugins/logstash-codec-netflow/tree/plugin-api-v1. We can then release a new  branch to support LS 2.x.

**Comment by jorritfolmer** [comment]
Yes backport just committed to the `plugin-api-v1` branch

**Comment by zamog** [comment]
Also waiting for the IPFIX support under the 2.1.0 version (for testing with Cisco NBAR2 and Performance monitor features data export) any way to install it without building the gem ?
i tried to run `bin/logstash-plugin install --version 2.1.0 logstash-codec-netflow` and get " version 2.1.0 does not exist"

**Comment by untergeek** [comment]
@zamog please open a separate issue for that request, as it should be tracked separately.  @jorritfolmer if you have pushed an update to rubygems, please close this issue.

**Comment by jorritfolmer** [comment]
Sorry but I don't have permission to push to rubygems with my account:

```
$ gem push logstash-codec-netflow-2.1.0.gem 
Pushing gem to https://rubygems.org...
You do not have permission to push to this gem.
```

**Comment by suyograo** [comment]
Published 2.1.0 for LS 2.x and 3.1.0 for LS 5.x. @jorritfolmer added you to RubyGems.org
